src/panels/config/entities/ha-config-entities.ts

```diff
@@ -453,6 +472,31 @@ export class HaConfigEntities extends SubscribeMixin(LitElement) {
                 >
               </div>`
             : ""}
+          ${this._numHiddenEntities && !activeFilters
```

Comment:
Can we show the hidden entities also when a filter is active, but in the same `<div>`? Got to be careful it doesn't get too big, especially on mobile

Reply:
I assume then without the "Show All" button?

Reply:
Yeah, not sure if it will work at all, but would be nice to also show the hidden if you have a filter on integration.

Reply:
The number of hidden entities should then still be global or only taking into consideration the hidden entities that match the active filter?

Reply:
match the active filter.

Reply:
Makes sense. I just adjusted the coding accordingly. See new screenshots.